Setup.md

```diff
@@ -0,0 +1,7 @@
+# Development Environment Setup
+- Install Anaconda on server
+- Create a new environment for Waldo under `models/env/`, this has location has already been set in the `.gitignore`
+- Install 
+  - pandas
+  - requests
+  - validators
```

Comment:
Do you have a requirements.txt that you can export? Would be alot easier create an environment from that with package versions etc.